### PR TITLE
Some function stuff

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/server/function/CommandFunctionManager.mapping [hunk 1]
```diff
@@ -1,40 +1,83 @@
...
+	FIELD field_33543 execution Lnet/minecraft/class_2991$class_6345;
+		COMMENT The active execution within this manager.
 	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/class_5349;)V
 		ARG 1 server
 		ARG 2 loader
-	METHOD method_12899 getTaggedFunctionSource ()Lnet/minecraft/class_2168;
+	METHOD method_12899 getCommandSource ()Lnet/minecraft/class_2168;
+		COMMENT {@return the command source to execute functions}
+		COMMENT
+		COMMENT <p>This command source {@linkplain ServerCommandSource#hasPermissionLevel(int)
```
Comment (apple502j) [inline mappings/net/minecraft/server/function/CommandFunctionManager.mapping:29]: Is this affected by function-permission-level in server.properties?

Comment (liach) [reply mappings/net/minecraft/server/function/CommandFunctionManager.mapping:29]: That is only used when parsing functions. I need to look into brigadier more for how exactly it works.

**Comment by liach** [comment]
Hmm, so no one reviews this? This at least maps a nested class...

**Review by modmuss50 — APPROVED** [review]
LGTM, will merge. 👍

**Pull request merged**